lms/djangoapps/teams/tests/test_views.py

```diff
@@ -777,6 +785,40 @@ def test_access_by_username(self, user, status, has_content):
             else:
                 self.assertEqual(membership['count'], 0)
 
+    @ddt.data(
+        ('student_enrolled_both_courses_other_team', 'TestX/TS101/Test_Course', 200, 'Nuclear Team'),
+        ('student_enrolled_both_courses_other_team', 'MIT/6.002x/Circuits', 200, 'Another Team'),
+        ('student_enrolled', 'TestX/TS101/Test_Course', 200, u'sólar team'),
+        ('student_enrolled', 'MIT/6.002x/Circuits', 400, ''),
+    )
+    @ddt.unpack
+    def test_course_filter_with_username(self, user, course_id, status, team_name):
```

Comment:
Normally, I agree. In this case in particular, it should read: 'When we test with this user and this course, we expect this status, and if there's a response it will be this team"